### ISSUE TITLE
The limits of chromosome

### ISSUE THREAD

**Issue by ZhangZhenYang-zzy** [dleelab/dist#1]
Could you please make the number of chromosome limited between 1 and 30?

**Comment by dleelab** [comment]
Hi Zhang,

You can use an updated version of DIST/DISTMIX in the GAUSS R package. This
version uses 33KG reference panel data containing ~33K subjects. Here is
the link:
Please let me know if you have any questions.

https://github.com/statsleelab/gauss

Best,
Don

**Comment by ZhangZhenYang-zzy** [comment]
Thanks for your response. However, I want to use the dist to impute the gwas summary data of cow, and the cow has 29 autosomes, but the DIST/DISTMIX limits the chromosome between 1 and 22, could you please change the limit between 1 and 30? Thank you so much!

**Comment by dleelab** [comment]
Unfortunately, DIST/DISTMIX only can be used for human data because it uses
1000 human Genome Projects data as a reference panel for now. If the cow's
whole genome sequencing data sets are available, we can make a DIST/DISTMIX
reference panel.

**Comment by ZhangZhenYang-zzy** [comment]
Yes, I have made a cow's reference panel by myself, and it worked. And I think DIST is fast and convenient， so if possible， could you please provide another version of DIST, and it doesn't limit the chromosome ?

**Comment by bacanusa** [comment]
Hi Zhenyang,

I think Chris (cc-ed here) has some later versions.
The chromosome/chromosome arm is specified only to run it in parallel faster.

Best,
Silviu-Alin Bacanu
Stat Gen, VIPBG
Dept of Psychiatry, Virginia Commonwealth University
Zoom: https://vcuhealth.zoom.us/j/8048287553
[signature_1346492113][signature_906049955]
--


From: zhangzhenyang ***@***.***>
Date: Monday, July 25, 2022 at 11:58 AM
To: dleelab/dist ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] Re: [dleelab/dist] The limits of chromosome (Issue #1)

Yes, I have made a cow's reference panel by myself, and it worked. And I think DIST is fast and convenient， so if possible， could you please provide another version of DIST, and it doesn't limit the chromosome ?
